sound/soc/sof/sof-pci-dev.c

```diff
@@ -284,6 +284,23 @@ static const struct sof_dev_desc jsl_desc = {
 };
 #endif
 
+#if IS_ENABLED(CONFIG_SND_SOC_SOF_ALDERLAKE)
+static const struct sof_dev_desc adls_desc = {
+	.machines		= snd_soc_acpi_intel_hda_machines,
+	.resindex_lpe_base      = 0,
+	.resindex_pcicfg_base   = -1,
+	.resindex_imr_base      = -1,
+	.irqindex_host_ipc      = -1,
+	.resindex_dma_base      = -1,
+	.chip_info = &adls_chip_info,
+	.default_fw_path = "intel/sof",
+	.default_tplg_path = "intel/sof-tplg",
+	.default_fw_filename = "sof-adl-s.ri",
+	.nocodec_tplg_filename = "sof-adl-nocodec.tplg",
```

Comment:
do we also need to use `sof-adl-s-nocodec.tplg` here? If there is any different clock setting and SSP configuration with different ADL platforms.

Reply:
@xiulipan I considered this, but for now we have no additional needs to variate, so I propose to stick to similar approach than we had with TGL and just have one nocodec file for the family.